### ISSUE TITLE
RESTEasy Reactive extension does not link to its guide

### ISSUE THREAD

**Issue by edeandrea** [quarkusio/code.quarkus.io#581]
On code.quarkus.io, the RESTEasy Reactive extension in the extension list does not have a link to [its guide](https://quarkus.io/guides/resteasy-reactive). See screenshot:

![Screen Shot 2021-10-05 at 8 02 49 AM](https://user-images.githubusercontent.com/363447/136018906-3f6f4c75-6033-49dc-83fc-30ffa678caf3.png)

**Comment by ia3andy** [comment]
good catch @edeandrea, this should be fixed in the extension metadata on Quarkus core..
